CHANGELOG.md

```diff
@@ -15,6 +15,23 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.8.3
```

Comment:
Should probably go to 5.9.0 since we added a new flag (and maybe because we're breaking an experimental feature?)

Reply:
```suggestion
# 5.9.0
```